src/managers/config.py

```diff
@@ -538,7 +538,7 @@ def external_listeners(self) -> list[Listener]:
                 Listener(
                     auth_map=auth,
                     scope="EXTERNAL",
-                    host=self.state.unit_broker.host,
+                    host=self.state.unit_broker.node_ip,
```

Comment:
That is a chance `node_ip` is an empty string. What happens in this case?

[This line will end up being `<listener>://:<port>`](https://github.com/canonical/kafka-k8s-operator/blob/7e4cfa5e19a7b87572cb6ee39eb8e899e43bb2a8/src/managers/config.py#L136-L139). AFAIR that is not an issue on Kafka, but I wanted to confirm it with you.